install/updateVersion.sh

```diff
@@ -161,15 +161,16 @@ export CA_HUB="${CA_HUB}"
 export CA_TAG="${CA_TAG}"
 export MIXER_HUB="${MIXER_HUB}"
 export MIXER_TAG="${MIXER_TAG}"
-export ISTIOCTL_URL="${ISTIOCTL_URL}"
 export PILOT_HUB="${PILOT_HUB}"
 export PILOT_TAG="${PILOT_TAG}"
+export ISTIOCTL_URL="${ISTIOCTL_URL}"
 export PROXY_TAG="${PROXY_TAG}"
 export ISTIO_NAMESPACE="${ISTIO_NAMESPACE}"
 export AUTH_DEBIAN_URL="${AUTH_DEBIAN_URL}"
 export PILOT_DEBIAN_URL="${PILOT_DEBIAN_URL}"
 export PROXY_DEBIAN_URL="${PROXY_DEBIAN_URL}"
-
+export FORTIO_HUB="${FORTIO_HUB}"
```

Comment:
Small comment - do we need so many hubs ? Also for fortio I think it may be ok to just use ':latest' and simplify things...

(just IMHO)

Reply:
I agree it's a lot overall but I'm just following the pattern in the code where things are split by hub and tag like all the other components. a follow up PR will make the tag auto updated with the rest